wrapper/src/main/java/software/amazon/jdbc/ds/AwsWrapperDataSource.java

```diff
@@ -67,6 +69,8 @@ public class AwsWrapperDataSource implements DataSource, Referenceable, Serializ
   private static final String SERVER_NAME = "serverName";
   private static final String SERVER_PORT = "serverPort";
 
+  private static final StorageService storageService = new StorageServiceImpl();
```

Comment:
We have another instance of the service in Driver class. We should use a single one.

Reply:
The Driver and AwsWrapperDataSource are independent classes so there is not a way to pass the storage service from one to the other. The Driver class needs the storage service reference so that it can clear it when Driver.clearCaches is called. I think the only way we could make sure its a single instance is if we make it a singleton, do we want to do that?